_config.yml

```diff
@@ -1,3 +1,9 @@
 title: JupyterHub and BinderHub Helm charts for Kubernetes
 description: This repository stores Helm chart tarballs for BinderHub and JupyterHub.  Actual development of the Helm charts takes place in the BinderHub and Zero to JupyterHub repos.
 
+theme: jekyll-theme-minimal
+
+repository: jupyterhub/zero-to-jupyterhub
```

Comment:
```suggestion
repository: jupyterhub/zero-to-jupyterhub-k8s
```

Reply:
I assume this is a typo. If this refers to readthedocs, then this is correct, if it refers to github, it is wrong.